fiftyone.pipeline.cloudrequestengine/tests/requestClient.test.js

```diff
@@ -48,11 +41,14 @@ const getTest = function (origin, writeResponse, checkRequest, checkResponse, do
     writeResponse(res);
   };
 
-  server = http.createServer(requestListener);
-  server.listen(3000, () => {
-    const response = client.get('http://localhost:3000', origin);
+  let server = http.createServer(requestListener);
+  server.listen(INITIAL_PORT, () => {

```

Comment:
can we maybe specify explicitly `3000` here, and `3001` in the other test without this ugly hack of incrementing the constant :)